code/modules/cargo/centcom_podlauncher.dm

```diff
@@ -514,4 +517,26 @@ force_open = FALSE, datum/tgui/master_ui = null, datum/ui_state/state = GLOB.adm
 	updateCursor(FALSE) //Make sure our moues cursor resets to default. False means we are not in launch mode
 	qdel(temp_pod) //Delete the temp_pod
 	qdel(selector) //Delete the selector effect
-	. = ..()
\ No newline at end of file
+	. = ..()
+
+/datum/centcom_podlauncher/proc/supplypod_punish_log(var/list/whoDyin)
+	var/podString = effectBurst ? "5 pods" : "a pod"
+	var/whomString = ""
+	if (!isemptylist(whoDyin))
```

Comment:
Mind dropping this helper and just doing direct list check + null check / lazylen